embroider-bug/rollup.config.mjs

```diff
@@ -14,12 +14,12 @@ export default {
   plugins: [
     // These are the modules that users should be able to import from your
     // addon. Anything not listed here may get optimized away.
-    addon.publicEntrypoints(['components/**/*.ts']),
```

Comment:
all of these paths are in terms of _outputs_, not inputs -- so js is the thing to use